open-craft/opencraft
# [OC-5197] Fix copyright years

This PR updates the copyright notice in OCIM's footer to automatically display the current year (cf. [`templates/footer.html`](https://github.com/open-craft/opencraft/compare/fix-years?expand=1#diff-c66cb68f1eee4aa7612734360a731340)).

It also updates copyright years at the top of each source file.

**Test instructions**

1. Access OCIM stage.
1. Log out if you're currently logged in.
1. Scroll to the bottom of the page and observe that the year mentioned in the footer is 2018.

**Reviewers**

- [x] @ahmedaljazzar
